devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusInfo.java

```diff
@@ -34,6 +34,9 @@ public QuarkusInfo() {
 
     @TaskAction
     public void logInfo() {
+
+        getProject().getLogger().warn("This task is experimental, its options and output might change in future versions");
```

Comment:
```suggestion
        getProject().getLogger().warn("quarkus info is experimental, its options and output might change in future versions");
```

In maven output this comes possibly many many lines away from what you just executed thus better to be explicit.